### PR TITLE
[Feature] Added support for `force` option when sending flow mods

### LINKED ISSUE #46: Introduce a `force` option when pushing flow mods (issue 26 part 2)
This is for the second part of the requirement mentioned on issue #26, breaking this down to simplify the pull request and its validations, tests and dependencies (this also depends on issue #2 for error handling propagation):

`2) adds an extra option force that would persist the request (to be sent to the switches later by the consistency routine).`

### PR DESCRIPTION
**Pull request by viniarck**
Fixes #46 

### Description of the change

- Added support for ``force`` option when sending flow mods

2021-11-17-152552_930x697_scrot

Once this change lands, clients can decide reliably if they want to ``force`` the flow mode to be sent later on (via consistency check) in case any switch connection errors happened as the flows were being installed. So, it's up to each client to decide if the client wants the flows to be either forced or not, for example, if `mef_eline` wants the flows being installed or fail immediately in case a switch isn't connected, then set `force: false`, for applications that don't care when the flow should be installed, and there's no other potential paths that the client could make of use then setting `force: true` would be handy. 

Notice that errors could still happen even when `force: true` is set (for instance, imagine a client sends a flow mod with force true, the switch isn't available and then the installation has an unsupported instruction for that switch), the rest of error propagation, and more tests for this, will be covered on issue #2.

### CODE REVIEW

**Review by ajoaoff — APPROVED** [review]

**Comment by viniarck** [comment]
thanks for reviewing @ajoaoff, I'll merge this once PR #49 lands since it's on top of it.

**Review by italovalcy — APPROVED** [review]
Looks good to me!

**Comment by viniarck** [comment]
Appreciated the reviews, guys.

**Pull request merged**
